src/SFA.DAS.AANHub.Application/Profiles/Queries/GetProfiles/GetProfilesQueryValidator.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FluentValidation;
+using SFA.DAS.AANHub.Domain.Common;
+
+namespace SFA.DAS.AANHub.Application.Profiles.Queries.GetProfiles
+{
+    public class GetProfilesQueryValidator : AbstractValidator<GetProfilesByUserType>
+    {
+        public GetProfilesQueryValidator()
+        {
+            RuleFor(a => a.UserType)
+                .NotEmpty()
+                .IsEnumName(typeof(UserType), caseSensitive: false);
```

Comment:
Do we need a test for this? I can see that the UserType None is not included in the TestCase

Reply:
I have added testcase for "None" usertype in GetProfilesQueryValidatorTests however, need to check with others if we want to implement it this way or validate it in QueryValidator to avoid executing further for this usertype

Reply:
I am happy for this PR to proceed to Test. We can look at any additional tests later